### ISSUE TITLE
blocked from libreview.io

### ISSUE THREAD

**Issue by tompusey** [FokkeZB/libreview-unofficial#5]
I've been using everything fine up until recently. I have an esp32 that uses an e-ink screen to get my blood glucose to display it every 10 minutes. Within the last couple of days it's stopped working and by testing what happens when it runs, I've discovered the following message from submitting a request to the API.

Sorry, you have been blocked
You are unable to access libreview.io

Any tips on getting myself unblocked?

**Comment by FokkeZB** [comment]
@tompusey o wow, I don't hope they started blocking accounts that try to use their private API 😨 

Did this also break LibreLinkup for you?

**Comment by tompusey** [comment]
Libre linkup works fine on my mobile (both on mobile data and on WiFi) and XDrip on my mobile, which uses a web follower (I'm guessing that uses the same or similar API? Not sure about that one) still works. Just the ESP32 and my laptop when I try using the API through https://libreview-unofficial.stoplight.io both throw up this error

**Comment by FokkeZB** [comment]
@tompusey one thing that could cause this may be the `version` header. That should match the version of the LibreLinkUp app you are using, or else at some point they will stop allowing the version to make request. Could you try bumping that?

**Comment by tompusey** [comment]
Tried going from 4.7 to what my phone has (4.9) and I'm still getting the same thing
![image](https://github.com/FokkeZB/libreview-unofficial/assets/88928988/e0873b84-78be-4c1c-bd01-a7c9bc5b35e9)

**Comment by AshCusack** [comment]
Yep, same issue here. stopped working on the 28th. I do get a different result on a browser and through a CURL request/postman request. I suspect they have changed the REST path, the domain itself is still the same according to DNS request my phone makes.

**Comment by AshCusack** [comment]
**Potential Fix:** Spent an hour or so tracking this down, sending a string for the "User-Agent" in the header has fixed this for me.
I'm using PHP and CURL, this is what i've got (excluding tokens etc.)
`CURLOPT_HTTPHEADER => array(
    'product: llu.android',
    'version: 4.9',
    'authorization: Bearer XXX',
    'Cookie: XXX',
    'User-Agent: Testing'
  ),`

**Comment by FokkeZB** [comment]
Thanks @AshCusack - so it seems User-Agent is now required.

I'll see if I can make time to work that into the docs.

**Comment by tompusey** [comment]
Thanks @AshCusack and @FokkeZB, added the user agent bit to my esp32/e-ink thing and it's back up and running as before.
